fhir-parser-resources/template-unittest.py

```diff
@@ -1,21 +1,16 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-#
-#  Generated from FHIR {{ info.version }} on {{ info.date }}.
-#  {{ info.year }}, SMART Health IT.
-
+#  Generated from FHIR {{ info.version }}, SMART Health IT.
```

Comment:
I changed this part of the test template so that as we make small tweaks, we don't get a million comment changes as a result.

I figured it was still reasonable to have it for the actual shipped models? But for the tests, meh.